django/views/csrf.py

```diff
@@ -97,12 +97,11 @@
 """
 
 
-def csrf_failure(request, reason=""):
+def csrf_failure(request, reason="", template_name="403_csrf.html"):
```

Comment:
@carljm that's exactly what I had in mind when I suggested using `None` as a default value.